### PR TITLE
Fix for <cell> inlineStr attribute without <si> child.

### PR DESCRIPTION
**Pull request by jmcnamara**
Fix for issue #75.

### CODE REVIEW

**Comment by cjw296** [comment]
@jmcnamara - looks good, any chance you could rebase? Someone else has been asking for this...

**Comment by jmcnamara** [comment]
@cjw296 Will do. I'm off line for the next 10 days but I'll do it once I'm back.

**Comment by jmcnamara** [comment]
I've rebased the patch and test against the master. Let me know if it needs any rework.

File: xlrd/xlsx.py [hunk 1]
```diff
@@ -717,17 +717,23 @@ def bad_child_tag(child_tag):
...
+                        tvalue = child.text
+                    elif child_tag == F_TAG:
+                        formula = child.text
                     else:
                         bad_child_tag(child_tag)
-                assert tvalue is not None
-                self.sheet.put_cell(rowx, colx, XL_CELL_TEXT, tvalue, xf_index)
+                if not tvalue:
+                    if self.bk.formatting_info:
+                        self.sheet.put_cell(rowx, colx, XL_CELL_BLANK, '', xf_index)
```
Comment (cjw296) [inline xlrd/xlsx.py]: bit disappointed to see no assertions involving XL_CELL_BLANK...
Will merge anyway, but can you add another test case?

**Pull request merged**